pkg/codec/splunkhec/splunk_to_logdata.go

```diff
@@ -87,11 +84,11 @@ func splunkHecToLogData(logger *zap.Logger, events []*splunk.Event, resourceCust
 	return ld, nil
 }
 
-// splunkHecRawToLogData transforms raw splunk event into log
-func splunkHecRawToLogData(sc *bufio.Scanner, query url.Values, resourceCustomizer func(pcommon.Resource), config *Config) (plog.Logs, int) {
+// SplunkHecRawToLogData transforms raw splunk event into log
+func SplunkHecRawToLogData(sc *bufio.Scanner, query url.Values, resourceCustomizer func(pcommon.Resource), config *splunk.HecToOtelAttrs) (plog.Logs, int) {
```

Comment:
This is a naming stutter which goes against the golang style guide. 

```suggestion
// RawToLogData transforms raw splunk event into log
func RawToLogData(sc *bufio.Scanner, query url.Values, resourceCustomizer func(pcommon.Resource), config *splunk.HecToOtelAttrs) (plog.Logs, int)
 ```

Reply:
hmm indeed and maybe that doesn't need to be extracted. I'll bring it back for now.